common/styles/utilities/_root-scope-classes.scss

```diff
@@ -486,6 +486,10 @@
   @include clearfix;
 }
 
+.uppercase {
+  text-transform: uppercase;
```

Comment:
I've got a feeling this exists already?

Reply:
https://github.com/wellcometrust/wellcomecollection.org/blob/master/common/styles/utilities/_root-scope-classes.scss#L91-L92